tests/tests-downloads.php

```diff
@@ -150,8 +108,8 @@ public function test_edd_download() {
 		$this->assertNotEmpty( $download3->prices );
 		$this->assertEquals( $prices, $download3->prices );
 		$this->assertEquals( $prices, $download3->get_prices() );
-		$this->assertEquals( 59, $download3->sales );
-		$this->assertEquals( 129.43, $download3->earnings );
```

Comment:
I increased the earnings for this download specific for testing (and also a value below).